### DISCUSSION TITLE
Pranidhi10 is still active! (needs off topic category)

### DISCUSSION THREAD

**Discussion by Rubiidev-IPad** [cytrinc/bark-coding#191] [Admin General]
![IMG_1076](https://github.com/mariocraft987/bark-coding/assets/169850052/f9b3035d-7f8f-44ba-8133-a27ad1949f42)
she still needs to do more work on bark tho

**Comment by raynepaws** [comment]
they're not a member of the bark team, but it was nice of them to make a cool theme for us

**Reply by mariocraft987** [reply]
bruh, shes only a contributer

**Comment by nmsderp** [comment]
> bruh, shes only a contributer
